variables.tf

```diff
@@ -90,6 +90,12 @@ variable "allow_rebase_merge" {
   default     = null
 }
 
+variable "allow_update_branch" {
+  description = "(Optional) Set to true to suggest updating pull request branches. (Default: false)"
+  type        = bool
+  default     = null
```

Comment:
nit

```suggestion
  default     = false
```

Reply:
@shinenelson, the default on a bool is false in the first place, so we either strip away all not needed `default = null` from bools that are false anyway or spec every default to false explicitly.